pkg/machines/libvirt-dbus.js

```diff
@@ -735,6 +752,19 @@ LIBVIRT_DBUS_PROVIDER = {
         };
     },
 
+    RESUME_VM({
+        name,
+        connectionName,
+        id: objPath
+    }) {
+        return dispatch => {
+            call(connectionName, objPath, 'org.libvirt.Domain', 'Resume', [], TIMEOUT)
+                    .fail(exception => dispatch(vmActionFailed(
```

Comment:
Let's please use the standard promise `.except()` instead of the cockpit specific `.fail()`.

Reply:
Just to be sure, you meant `.catch()`, right?

Reply:
*blush*, of course :-)